image_tools/test/cam2image.txt

```diff
@@ -0,0 +1,2 @@
+Publishing data on topic 'image'
+Publishing image #1
```

Comment:
Can you give more context about why renaming this file was necessary ? Asking because I thought that launch testing was explicitely looking for `.regex` files